src/SelectTeamPicker.ts

```diff
@@ -0,0 +1 @@
+export * from "./controls/SelectTeamPicker";
```

Comment:
Would it be better to rename the controls to `TeamPicker`? In that case the name will be aligned with other pickers.